common/src/main/java/bisq/common/config/Config.java

```diff
@@ -577,6 +582,12 @@ public Config(String defaultAppName, File defaultUserDataDir, String... args) {
                         "Prevents mempool check of trade parameters")
                         .withRequiredArg()
                         .ofType(boolean.class)
+                        .defaultsTo(false);  
+        
+        ArgumentAcceptingOptionSpec<Boolean> apiUserOpt =
+                parser.accepts(ISAPIUSER, "Mode to run daemon, DESKTOP or WEBAPP")
```

Comment:
We should find an alternative to adding a startup flag.

Reply:
yep